R/utilities.R

```diff
@@ -465,11 +465,15 @@ broom_confint <- function(x, ...) {
 }
 
 # this version adds a terms column
-broom_confint_terms <- function(x, ...) {
+broom_confint_terms <- function(x, conf.method = "plik", ...) {
```

Comment:
Instead of making this change in `broom_confint_terms`, let's relocate this code into the `tidy.glm()` method.

Reply:
Revisiting and I agree with my prior point here -- this change should be local to `glm` and `lm` methods, rather than occurring in a general helper function.